translate.js

```diff
@@ -0,0 +1,80 @@
+const RUSSIAN = {
```

Comment:
Usually app translation consist of a lot of words and developer should split translation object (like `RUSSIAN`) into separate files. It simplifies development and debugging. Do nothing here just so you know